README.md

```diff
@@ -2,72 +2,67 @@
 
 # Project Name
 
-> Description the project.
+> Awesome books - Javascript with Objects
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+Feel free to check the: [Portfolio-link](https://ehs9nino.github.io/Awesome-books/)
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
```

Comment:
- [OPTIONAL] Please it will be nice to add explicit steps on how to clone and run your app locally, as this will be of help to non technical users and some of them are potential recruiters. Your `readme file` will be more informative and professional if you stick to this [template](https://github.com/microverseinc/readme-template) from Microverse.

Reply:
@Mihndim2020 will work on it!